samples/tutorials/hardwareInterfaces/gpio/TimerLed/timerLedComponent/timerLed.c

```diff
@@ -0,0 +1,66 @@
+//--------------------------------------------------------------------------------------------------
+/*
+ * This sample app is a blinking LED at 1 second interval
+ */
+//--------------------------------------------------------------------------------------------------
+
+#include "legato.h"
+#include "interfaces.h"
+
+#define LED_SAMPLE_INTERVAL_IN_SECONDS (1)
+
+static int8_t state = 0;
+
+//--------------------------------------------------------------------------------------------------
+/**
+ * Sets default configuration LED D750 as off
+ */
+//--------------------------------------------------------------------------------------------------
+static void ConfigureLed
+(
+    void
+)
+{
+    // Configure initial value as LED off
+    LE_FATAL_IF(mangoh_ledGpio_SetPushPullOutput(MANGOH_LEDGPIO_ACTIVE_HIGH, true) != LE_OK,
```

Comment:
The comment says that the initial LED state is off which leads me to believe that this LED is active low.  If that's the case, then why don't we configure the GPIO as `MANGOH_LEDGPIO_ACTIVE_LOW`?